warrior/WarriorCore/Classes/html_results_class.py

```diff
@@ -57,18 +57,17 @@ def set_attributes(self, line, variant, stepcount):
                      'type': variant.replace('Test', '').replace('Keyword', 'step ') + str(stepcount),
                      'name': line.get("name"),
                      'info': self.get_info(line),
+                     'description': line.get("description"),
                      'timestamp': line.get("timestamp"),
                      'duration': line.get("time"),
                      'status': '<span class=' + status_name + '>' + status_name + '</span>',
                      'impact': line.get("impact"),
                      'onerror': line.get("onerror"),
-                     'msc': '<span style="padding-left:10px; padding-right: 10px;"><a href="' + result_file
```

Comment:
Hi @priyaarun23,

What was 'msc' and why was it removed?

Thanks,
Sanika

Reply:
@sanikakulkarni ,

	'msc' is just the single value which has the appended values of Result, Log & Defect. It also returns a single value to a single column(HTML result summary table).
	
	Since I have split the columns as three(Result, Log & Defect), the single 'msc' value has also been split as Result, Log & Defect which in-turn return its corresponding three values in three different columns(HTML result summary table).
	
	As it is modularised now, modifications in future can be done easily.